### DISCUSSION TITLE
Does the Gitbook Google Analytics Integration support GA4 Google Tag IDs?

### DISCUSSION THREAD

**Discussion by MantisClone** [GitbookIO/community#316] [0. General bugs and questions] [answered]
Details:
* The documentation [Configure the Google Analytics integration](https://docs.gitbook.com/product-tour/integrations/google-analytics/configure) explicitly mentions "Tracking ID" aka. Universal Analytics (UA) properties. 
* Universal Analytics (UA) properties [stopped processing new data on July 1, 2023.](https://support.google.com/analytics/answer/11583528?sjid=5728426737950369377-NA)

**Comment by aikoii** [comment, marked as answer]
Hey @MantisClone! Yes — GitBook does support GA4, and a G-* ID should work just fine 🙂. I totally get why it can be a bit confusing, and I've made a note of this internally 👍.
